src/main/java/org/myrobotlab/service/ProgramAB.java

```diff
@@ -995,13 +982,17 @@ public Locale getLocale() {
 
   @Override
   public Map<String, Locale> getLocales() {
-	  // FIXME should be based on bots found ???
-   return Locale.getLocaleMap("en-US", "fr-FR", "es-ES", "de-DE", "nl-NL", "ru-RU", "hi-IN","it-IT", "fi-FI","pt-PT");
+    // FIXME should be based on bots found ???
+    return Locale.getLocaleMap("en-US", "fr-FR", "es-ES", "de-DE", "nl-NL", "ru-RU", "hi-IN", "it-IT", "fi-FI", "pt-PT");

```

Comment:
This method returns non-sense..  the list of locales that this instance of ProgramAB supports would be the list of locales for any of the bots that are currently available..  not this list.   make it return null.. or something.. this list is just wrong.

Reply:
agreed - its nonsense ..
made it return the list of bot locales